apps/src/javalab/DisplayMode.js

```diff
@@ -0,0 +1,23 @@
+export const DisplayMode = {
```

Comment:
One NIT: DisplayTheme may be clearer. DisplayMode could imply view/edit mode or mobile/desktop mode.

Reply:
Good call, yep I agree. I can rename this.

Reply:
Updated to `DisplayTheme`/`display_theme`